src/context/ToolboxContext/index.tsx

```diff
@@ -26,6 +27,8 @@ const reducer = (state: ToolboxState, action: ToolboxAction): ToolboxState => {
       return { ...state, selectedCollection: action.payload };
     case 'SET_SELECTED_COLLECTION_ITEMS':
       return { ...state, selectedCollectionItems: action.payload };
+    case 'SET_SELECTED_COLLECTION_ITEM':
```

Comment:
Just a heads up for the future, but there will be a future ticket that is going to add complexity to how we manage state for STAC collections. 

As of right now the STAC search seems to be doing a root-level search (I'm not actually entirely sure), but a future ticket is going to introduce further nesting. So, this approach to state would still work if we don't have to store another nested collection(s) at the same time. 

It's fine for now, but if the time comes that displaying different nested catalogs/collections, we may have to give it some thought.